docs/standard/modelling/repr-trusts.rst

```diff
@@ -49,26 +48,26 @@ A trust with a known beneficiary, known trustee, known settlor and known protect
    :align: center
    
    
-Scenario 3
-^^^^^^^^^^
+Trust with combined roles and anonymous beneficiary 
```

Comment:
Person C is this diagram should be represented by the question mark in a circle icon.